### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by I-can-have-a-long-username-if-I-want-to** [THEREALWWEFAN231/Inertia#795]
**Minecraft Version(s)**
all
**What is the suggestion**
Add wall"mods"
Basically makes the walls transparent so that all entities and blocks on the other side visible. 
**Why would this feature be useful**
the ability to see which blocks are on the other side may be useful when navigating obsidian mazes and would allow the player to see which direction the other player is looking.

**Comment by MLGgamingpro** [comment]
The `"Chams"` module lets you see entities through walls. As for the other one you can use `"Camera Clip"` to extend your `F5` camera.

**Comment by I-can-have-a-long-username-if-I-want-to** [comment]
Yes, I know there are technically ways to achieve a similar result, but freecam is kinda odd to control if you want to see the blocks on the other side of a wall.

**Comment by feliscatusmeows** [comment]
can't call them wall h@(|<5
